test/performance/broker-rmq/README.md

```diff
@@ -0,0 +1,78 @@
+While https://github.com/knative-sandbox/eventing-rabbitmq/pull/445 contains the YAML that configures the benchmark, and we (+@ikvmw) also intend to have the step-by-step guide in that PR as well, I wanted to start with the WIP guide variant here, and move it across when it's done. Here it goes:
+
+## 1/5. Pre-requisites
+- GKE 1.20 with autoscaling and `c2-standard-4` machine type in the default nodepool
+- kubectl 1.20
+- [ko 0.9.3](https://github.com/google/ko/releases/tag/v0.9.3) + [`KO_DOCKER_REPO`](https://github.com/knative/eventing/blob/main/DEVELOPMENT.md#setup-your-environment)
+- A git clone of [knative-sandbox/eventing-rabbitmq](https://github.com/knative-sandbox/eventing-rabbitmq) at tag v0.27.0 or higher
+We use [GCP `c2-standard-4`](https://cloud.google.com/compute/vm-instance-pricing#compute-optimized_machine_types)  machine type to make benchmarks results consistent. Using a different instance type, or a different cloud provider, is almost certain to yield different results.
+## 2/5. Install Knative Serving
+We install a specific version of Knative - [v0.26.0](https://github.com/knative/eventing/releases/tag/v0.26.0) - as this is a point-in-time guide. While we expect subsequent versions to continue working the same way, in the absence of automated tests that ensure this, we stick to exact versions that we have tested manually.
+```
+# Installing Knative Serving ...
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.26.0/serving-crds.yaml
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.26.0/serving-core.yaml
```

Comment:
The latest version of this is here https://github.com/knative-sandbox/eventing-rabbitmq/issues/239#issuecomment-935716597, and I think that we should update this PR with that content.

Our goal was to keep the guide self-contained, meaning that the user would have everything in that single place. As it's a point-in-time example, the focus was on using exact versions. I just did a small change which takes into account the state of eventing-rabbitmq after this PR gets merged in.

I have also confirmed that this works in GKE 1.21. The reason why I wanted to make sure that this works in K8s 1.21 is because 1.20 (the version that we tested initially) enters maintenance mode in a few months - `2021-12-28` - and [end of life within 5 months](https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md#120).

Reply:
That's where automation comes in. As soon as we have that, and continuously test that latest released components work (I'm thinking cluster-operator and messaging-topology-operator), then we switch to testing with dev knative components, so that we catch issues **before** we release a new version.